nxdrive/gui/application.py

```diff
@@ -648,6 +652,17 @@ def _root_deleted(self) -> None:
             engine.reinit()
             engine.start()
 
+    def _send_crash_metrics(self) -> None:
+        for engine in self.manager.engines.copy().values():
+            if engine.remote and State.has_crashed:
+                if not State.crash_details:
+                    engine.remote.metrics.send({CRASHED_HIT: 1})
```

Comment:
Move metrics into a dict. And update that dict with `State.crash_details` if available. That will make the code more readable and skip the need for a if/else block code.